src/Resources/Resources/ChangeLog.md

```diff
@@ -19,7 +19,11 @@
 -->
 ## Upcoming Release
 
-## Version 1.10.0
+## Version 0.12.0
```

Comment:
I forgot whether we need to bump the module version during publishing module from this branch. If bump version is required, you should not add version here.

Reply:
I have updated the versions now with this commit , https://github.com/Azure/azure-powershell/pull/16247/commits/9a63d94a2ebddcebae062c398c38b6de66deb035

In this branch we dont bump version automatically, it is manual